ext/opentelemetry-ext-pymongo/src/opentelemetry/ext/pymongo/__init__.py

```diff
@@ -0,0 +1,81 @@
+# Copyright 2019, OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+The opentelemetry-ext-pymongo package allows tracing commands made by the
+pymongo library.
+"""
+
+from pymongo import monitoring
+
+from opentelemetry.trace import SpanKind
+from opentelemetry.trace.status import Status, StatusCanonicalCode
+
+DATABASE_TYPE = "mongodb"
+COMMAND_ATTRIBUTES = ["filter", "sort", "skip", "limit", "pipeline"]
+
+
+def trace_integration(tracer=None):
+    """Integrate with pymongo to trace it using event listener.
+       https://api.mongodb.com/python/current/api/pymongo/monitoring.html
+    """
+
+    monitoring.register(CommandTracer(tracer))
+
+
+class CommandTracer(monitoring.CommandListener):
+    def __init__(self, tracer):
+        self._tracer = tracer
```

Comment:
What happens if `tracer` is None?

Reply:
I think we can assume here that the tracer must never be None. But you are right that in the spirit of defensive programming we could add a check here and raise an exception (since this is initialization code, we may do so)